oak_functions/loader/src/lookup.rs

```diff
@@ -153,3 +153,24 @@ pub fn parse_lookup_entries<B: prost::bytes::Buf>(
     }
     Ok(entries)
 }
+
+/// Wrapper around a LookupData instance for testing.
+#[allow(dead_code)]
+pub struct LookupDataForTest {
+    pub lookup_data: LookupData,
+}
```

Comment:
AFAICT this wrapper is never actually used, only the inner LookupData instance? If so, why not have a function to create a LookupData instance for test directly?

Reply:
True. I was not sure if it is a good idea to add a method for testing to `LookupData`. Fixed.